src/async-wrap.cc

```diff
@@ -320,7 +320,7 @@ Local<Value> AsyncWrap::MakeCallback(const Local<Function> cb,
 
   Local<Function> pre_fn = env()->async_hooks_pre_function();
   Local<Function> post_fn = env()->async_hooks_post_function();
-  Local<Value> uid = Number::New(env()->isolate(), get_uid());
+  Local<Value> uid = Number::New(env()->isolate(), static_cast<double>(get_uid()));
```

Comment:
I would recommend against changing these things here, there would be quite a few conflicts with https://github.com/nodejs/node/pull/11883